From: Benjamin Drung <benjamin.drung@canonical.com>
Date: Mon, 12 Oct 2026 17:13:14 +0200
Subject: Revert correcting Debian codename Experimental to RC-Buggy

"experimenal" is the alias for the codename RC-Buggy (similar to
"unstable" being the alias for the codename Sid). This change breaks the
distro-info test case and dgit (Debian bug #1145095). So revert this
change for the Ubuntu stable release updates.

Forwarded: not-needed
---
 debian.csv        | 2 +-
 up-to-date        | 2 +-
 validate-csv-data | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/debian.csv b/debian.csv
index afad95a..a655eeb 100644
--- a/debian.csv
+++ b/debian.csv
@@ -20,4 +20,4 @@ version,codename,series,created,release,eol,eol-lts,eol-elts
 14,Forky,forky,2025-08-09
 15,Duke,duke,2027-08-01
 ,Sid,sid,1993-08-16
-,RC-Buggy,rc-buggy,1993-08-16
+,Experimental,experimental,1993-08-16
diff --git a/up-to-date b/up-to-date
index ad14f17..75db6d6 100755
--- a/up-to-date
+++ b/up-to-date
@@ -26,7 +26,7 @@ import sys
 from lib.tools import convert_date, get_csv_dict_reader, main
 
 ROLLING_FOREVER = {
-    "debian": ("sid", "rc-buggy"),
+    "debian": ("sid", "experimental"),
     "devuan": ("ceres", "experimental"),
     "elxr": (),
     "ubuntu": (),
diff --git a/validate-csv-data b/validate-csv-data
index f03ce72..0c2b4b6 100755
--- a/validate-csv-data
+++ b/validate-csv-data
@@ -111,7 +111,7 @@ _DERIVATIVES = {
             "freia": "forky",
             "gryphon": "duke",
             "ceres": "sid",
-            "experimental": "rc-buggy",
+            "experimental": "experimental",
         },
     )
 }
